build/Helix/GenerateTestProjFile.ps1

```diff
@@ -0,0 +1,304 @@
+Param(
+    [Parameter(Mandatory = $true)] 
```

Comment:
put [CmdLetBinding()] on this so you can pass -verbose for debugging (And then you can liberally sprinkle Write-Verbose throughout your script to help when authoring)